scripts/update-se-drivers.js

```diff
@@ -31,12 +31,17 @@ console.log(`Chrome Major Version - ${chromeMajorVersion}`);
 
 const chromeDriverUrl = `https://chromedriver.storage.googleapis.com/LATEST_RELEASE_${chromeMajorVersion}`;
 
-axios.get(chromeDriverUrl).then((response) => {
-  const chromeDriverVersion = response.data;
-  console.log(`Chrome Driver Version - ${chromeDriverVersion}`);
+const chromeDriverVersion = spawnSync('curl', [chromeDriverUrl], { encoding : 'utf8' });
```

Comment:
nit: can destructure `stdout` and `stderr` into variables:

```javascript
const {stdout: chromeDriverVersion, stderr: errorResponse} = spawnSync('curl', [chromeDriverUrl], { encoding : 'utf8' });
```

Reply:
Done!